src/vs/workbench/contrib/chat/electron-sandbox/tools/fetchPageTool.ts

```diff
@@ -57,16 +54,9 @@ export class FetchWebPageTool implements IToolImpl {
 			}
 		}
 
-		const result = await this._readerModeService.extract(valid);
-		// Right now there's a bug when returning multiple text content parts so we're merging into one.
-		// When that's fixed we can use the helper function _getPromptPartForWebPageContents.
-		const value = result.map((content, index) => localize(
-			'fetchWebPage.promptPart',
-			'Below is the main content extracted from the webpage ({0}). Please read and analyze this content to assist with any follow-up questions:\n\n{1}',
-			valid[index].toString(),
-			content
-		)).join('\n\n---\n\n');
-		return { content: [{ kind: 'text', value }] };
+		const contents = await this._readerModeService.extract(valid);
+		// TODO: Should we return a content for invalid URLs so that the indexes are aligned?
```

Comment:
Indicating to the model what happened is good, otherwise it can try to call it again or simply act confused to the user